tests/test_elasticsearch.py

```diff
@@ -281,6 +282,71 @@ def test_backup_vault_kms_filter(self):
         aliases = kms.list_aliases(KeyId=resources[0]['EncryptionAtRestOptions']['KmsKeyId'])
         self.assertEqual(aliases['Aliases'][0]['AliasName'], 'alias/aws/es')
 
+    def test_elasticsearch_cross_account(self):
+        session_factory = self.replay_flight_data("test_elasticsearch_cross_account")
+        p = self.load_policy(
+            {
+                "name": "elasticsearch-cross-account",
+                "resource": "elasticsearch",
+                "filters": [{"type": "cross-account"}],
+            },
+            session_factory=session_factory,
+        )
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
+        access_policy = json.loads(resources[0]['AccessPolicies'])
+        self.assertIn("*", [s['Principal'] for s in access_policy.get('Statement')])
```

Comment:
we also want to verify the cross-account annotation content here on the resource.

Reply:
added check of CrossAccount violation contents in last test